tests/components/update/test_recorder.py

```diff
@@ -0,0 +1,55 @@
+"""The tests for update recorder."""
+from __future__ import annotations
+
+from datetime import timedelta
+
+from homeassistant.components.recorder.models import StateAttributes, States
+from homeassistant.components.recorder.util import session_scope
+from homeassistant.components.update.const import (
+    ATTR_CURRENT_VERSION,
+    ATTR_IN_PROGRESS,
+    ATTR_RELEASE_SUMMARY,
+    DOMAIN,
+)
+from homeassistant.const import CONF_PLATFORM
+from homeassistant.core import HomeAssistant, State
+from homeassistant.setup import async_setup_component
+from homeassistant.util import dt as dt_util
+
+from tests.common import async_fire_time_changed, async_init_recorder_component
+from tests.components.recorder.common import async_wait_recording_done_without_instance
+
+
+async def test_exclude_attributes(
+    hass: HomeAssistant, enable_custom_integrations: None
+):
+    """Test update attributes to be excluded."""
+    await async_init_recorder_component(hass)
+    platform = getattr(hass.components, f"test.{DOMAIN}")
+    platform.init()
+    assert await async_setup_component(hass, DOMAIN, {DOMAIN: {CONF_PLATFORM: "test"}})
+    await hass.async_block_till_done()
+    state = hass.states.get("update.update_already_in_progress")
+    assert state.attributes[ATTR_IN_PROGRESS] == 50
+    await async_setup_component(hass, DOMAIN, {})
+
+    await hass.async_block_till_done()
+    async_fire_time_changed(hass, dt_util.utcnow() + timedelta(minutes=5))
+    await hass.async_block_till_done()
+    await async_wait_recording_done_without_instance(hass)
+
+    def _fetch_sun_states() -> list[State]:
```

Comment:
```suggestion
    def _fetch_states() -> list[State]:
```
This does not seem to be limited to sun?

Reply:
Thanks. Copy pasta